### PR TITLE
Assorted cleanups and simplifications

### PR DESCRIPTION
**Pull request by jcorbin**
On the way to #28 and #27

### CODE REVIEW

File: parser.js [hunk 1]
```diff
@@ -20,29 +20,32 @@
...
-	read_arg1len: 4,
-	read_arg2len: 5,
-	read_arg3len: 6,
-	read_csum: 7,
-	read_arg1: 8,
-	read_arg2: 9,
-	read_arg3: 10,
-	error: 255
-};
+/* jshint camelcase:false */
```
Comment (Raynos) [inline parser.js]: :+1: like this cleanup.

File: index.js [hunk 2]
```diff
@@ -151,6 +152,7 @@ TChannel.prototype.addPeer = function (name, connection) {
 	connection.remoteName = name;
 };
 
+/* jshint maxparams:5 */
```
Comment (Raynos) [inline index.js]: jshint cleanup looks legit, thank you.

File: index.js [hunk 3]
```diff
@@ -419,7 +421,7 @@ TChannelConnection.prototype.onIdentify = function (frame) {
 		return true;
 	}
 
-	this.logger.error('error: first req on socket must be identify');
+	this.logger.error('first req on socket must be identify');
```
Comment (Raynos) [inline index.js]: good call on prefix dropping.

File: index.js [hunk 4]
```diff
@@ -516,20 +533,11 @@ TChannelServerOp.prototype.onResponse = function (err, res1, res2) {
...
-	frame.header.type = types.req_complete_message;
-	frame.header.id = ++this.lastSentMessage;
-	frame.header.seq = 0;
-
-	this.outOps[frame.header.id] = new TChannelClientOp(
-		options, frame, this.channel.now(), callback);
-	this.pendingCount++;
-	return this.socket.write(frame.toBuffer());
-};
+function isError(obj) {
```
Comment (Raynos) [inline index.js]: :+1:

**Comment by Raynos** [comment]
This all lgtm. Feel free to merge @jcorbin

File: parser.js [hunk 5]
```diff
@@ -20,29 +20,32 @@
...
+
+var types = {};
+types.reqCompleteMessage = types.req_complete_message = 0x01;
```
Comment (jwolski) [inline parser.js]: why keep the snake_cased around?

Comment (Raynos) [inline parser.js]: Let's remove the snake case in a seperate PR as its a breaking change.

It seems unfitting to make a breaking change in a cleanup/jshint PR.

**Pull request merged**